frontend/src/components/form/base/EDatePicker.vue

```diff
@@ -55,30 +55,22 @@ export default {
   methods: {
     format (val) {
       if (val !== '') {
-        return this.$moment.utc(val, this.valueFormat, this.$i18n.locale).format('L')
+        return this.$date.utc(val, this.valueFormat).format('L')
```

Comment:
It makes no sense to use $i18n.locale explicitly here. The dayjs locale is set globally in `setLanguage`.

Reply:
I believe this way necessary in moment.js, but all the better if it's globally configurable in day.js

Reply:
Are you sure? https://github.com/ecamp/ecamp3/blob/07a931704b3430d520046616e22bf2811888fc77/frontend/src/components/form/base/ETimePicker.vue#L62-L63 The moment locale was also globally configurable https://github.com/ecamp/ecamp3/blob/57f534f9ea230c67f7955b97d695d387d9878914/frontend/src/plugins/store/lang.js#L21

Reply:
:man_shrugging: